bindings/python/pymongoarrow/__init__.py

```diff
@@ -28,10 +25,16 @@
     def _parse_version(version):
         return _LooseVersion(version)
 
-# TODO: PYTHON-2659
-on_rtd = os.environ.get('READTHEDOCS') == 'True'
-if not on_rtd:
+
+try:
+    from pymongoarrow.lib import libbson_version
+
     if _parse_version(libbson_version) < _parse_version(_MIN_LIBBSON_VERSION):
         raise ImportError(
             f"Expected libbson version {_MIN_LIBBSON_VERSION} or greater, "
-            f"found {libbson_version}")
+            f"found {libbson_version}"
+        )
+except ImportError:
+    import warnings
```

Comment:
Should this import be at the top level under where `import os` was?

Reply:
Fixed